cats_and_dogs.md

```diff
@@ -87,31 +87,32 @@ Another example featuring `Ragdoll` and `basset_hound` is shown below:
 
 ## Task
 
-Your task is to build a service that exposes an API to return three properties of the images:
+Your task is to build a model that returns three properties of an input image:
 
-1. What animals are present in the image? A cat or a dog or both
-2. Identify the breeds of the animals present in the image. 
+1. What animals are present in the image? A cat or a dog or both.
+2. Identify the breeds of the animals present in the image.
 3. Provide a binary mask presenting which pixel features a pet (cat or dog) and which does not.
 
-We won't prescribe the API specification or expected response structure. We would like to see your approach to designing this API. 
+We won't prescribe the output specification or expected response structure – we would like to see your approach.
 
 ### Bonus 
 
 There will be some brownie points if you can write a short blurb in your README.md on how you would scale your solution to:
 
-a) Cater for images with more than 2 cats and dogs of all combinations of all breeds of pets we have seen in this dataset.
-b) If you were to scale your service to meet the 8K requests per second rate of user load.
+a. Expose a REST API and integrate your predictive model into a backend system.
+b. Cater for images with more than 2 cats and dogs of all combinations of all breeds of pets we have seen in this dataset.
+c. Scale your service to meet ~8,000 requests per second.
```

Comment:
**thought:** Should we add an option for them to add something they think is cool and useful but we don't have listed? This could be something they pick based on their prior experience/strength. It might be too broad, however.